api/envoy/config/core/v3/health_check.proto

```diff
@@ -145,6 +145,19 @@ message HealthCheck {
     // <envoy_v3_api_msg_type.matcher.v3.StringMatcher>`. See the :ref:`architecture overview
     // <arch_overview_health_checking_identity>` for more information.
     type.matcher.v3.StringMatcher service_name_matcher = 11;
+
+    enum Method {
+      GET = 0;
+      HEAD = 1;
+      POST = 2;
+      PUT = 3;
+      DELETE = 4;
+      OPTIONS = 5;
+      TRACE = 6;
+    }
+    
+    // HTTP Method that will be used for health checking, default is "GET"
```

Comment:
It would be helpful if we add more comments eg. if we use other HTTP methods, the `expected_statuses` could need to be set.

Reply:
nit: add '.' at the end of the first line.